roles-in-tech/entrepreneurship.md

```diff
@@ -29,6 +29,10 @@ No Prerequisites.
 
 ### Lesson
 
+* **References**: 
+  - [wikipedia](https://en.wikipedia.org/wiki/Entrepreneurship)
+  - [entrepreneur.com](https://www.entrepreneur.com/article/291082)
+
```

Comment:
Are you sure you don't need to cite Motivation, Common Misconceptions, or Guided Practice in the outline?

Reply:
Yes

Reply:
An awful lot of those sections match websites for several sentences in a row:
https://smallseotools.com/view-report/d8c0b2cf553092f4e9a34896b25632f0
https://smallseotools.com/view-report/adfb40a59a4f21700bd331b73adece7c/

Please cite them quickly with the website after the paragraph if this is where you originally got them. ([likethis]())

Reply:
ok got it. I was checking wrong file. i will add it.

Reply:
Is it good now?

Reply:
Looks good! Add guided practice too please, then I think you’ll be done.

Reply:
Done.